src/Polly.Core/README.md

```diff
@@ -57,14 +61,15 @@ exposed on this class that cover different scenarios:
 - Asynchronous void methods.
 - Asynchronous methods with result.
 
-For example, synchronous `Execute` method is implemented as:
+For example, the synchronous `Execute` method is implemented as:
 
 ``` csharp
 public void Execute(Action execute)
 {
     var context = ResilienceContext.Get();
+
     context.IsSynchronous = true;
-    context.IsVoid = true;
+    context.ResultType = typeof(VoidResult);
```

Comment:
In a follow-up PR, I wonder if we should integrate something like [MarkdownSnippets.Tool](https://www.nuget.org/packages/MarkdownSnippets.Tool) into the repo and have real working/compilable samples in the test project that can auto-generate so these things are easy to keep up-to-date and working as v8 gets implemented?

Reply:
Looks really nice and it would help to keep the docs in sync with the code. Love the idea. 

Fyi, we are also using `docfx` that supports the similar thing such as referencing snippets from the code using regions:
https://dotnet.github.io/docfx/docs/markdown.html?tabs=linux%2Cdotnet#code-snippet

I'll go ahead and merge this and we address this in follow-up.